app/uk/gov/hmrc/transitmovementsrouter/services/EISMessageTransformers.scala

```diff
@@ -31,27 +31,52 @@ import uk.gov.hmrc.transitmovementsrouter.services.state.UnwrappingState
 import uk.gov.hmrc.transitmovementsrouter.services.state.UnwrappingState.FoundMessageType
 import uk.gov.hmrc.transitmovementsrouter.services.state.UnwrappingState.LookingForMessageTypeElement
 import uk.gov.hmrc.transitmovementsrouter.services.state.UnwrappingState.LookingForWrappedElement
+import uk.gov.hmrc.transitmovementsrouter.services.state.WrappingState
 
 @ImplementedBy(classOf[EISMessageTransformersImpl])
 trait EISMessageTransformers {
 
+  def wrap: Flow[ByteString, ByteString, _]
+
   def unwrap: Flow[ByteString, ByteString, _]
 
 }
 
 @Singleton
 class EISMessageTransformersImpl extends EISMessageTransformers {
 
+  private val WRAPPED_MESSAGE_TYPE_PREFIX = "txd"
+  private val WRAPPED_MESSAGE_ROOT_PREFIX = "n1"
+  private val UNWRAPPED_PREFIX            = "ncts"
+  private val NCTS_NAMESPACE_URL          = "http://ncts.dgtaxud.ec"
+  private val XSI_NAMESPACE_URL           = "http://www.w3.org/2001/XMLSchema-instance"
+  private val EIS_NAMESPACE_URL           = "http://www.hmrc.gov.uk/eis/ncts5/v1"
```

Comment:
I Think we can move all this to Constants to make this class more clean and clear.

Reply:
I disagree. These constants are only relevant here so I think they should remain here.